homeassistant/components/webostv/media_player.py

```diff
@@ -68,6 +68,7 @@
 
 MIN_TIME_BETWEEN_SCANS = timedelta(seconds=10)
 MIN_TIME_BETWEEN_FORCED_SCANS = timedelta(seconds=1)
+PARALLEL_UPDATES = 0
```

Comment:
Just to double check, the integration is able to process multiple actions at the same time?

Reply:
Yes, upstream every command to the TV has unique id which is sent on the request and response:
https://github.com/home-assistant-libs/aiowebostv/blob/a160d9182053f602372df79e6be66f06cd6c6a00/aiowebostv/webos_client.py#L612